### DISCUSSION TITLE
带有复选的 Table 如何手动取消选择

### DISCUSSION THREAD

**Discussion by yangjinjun3** [ant-design/ant-design#35670] [General]
# 使用的是带有复选的Table，我的Table外有个「取消选择」的按钮，想实现通过点击取消选择，把Table中已选择的清除掉，如何实现？

![This is an image](https://user-images.githubusercontent.com/44964193/169677956-37991069-8cba-4bf8-84ff-934f833b6fcc.png)

**Comment by yoyo837** [comment]
给selection搞受控模式

**Reply by yangjinjun3** [reply]
不明白，能否详细举例？

**Reply by yangjinjun3** [reply]
您好，对于selection的受控模式不是很熟悉，能否给个大概示例，急用，另外还有下图的全选按钮，也帮忙看下，多谢。


<img width="333" alt="image" src="https://user-images.githubusercontent.com/44964193/170158878-e57559dc-3590-461e-b3c4-b84cd9ef5ee8.png">
